src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-application-migration-services-integration.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: AWS Application Migration Service integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+  - Application Migration
+metaDescription: "New Relic's Amazon Application Migration Service integration"
+freshnessValidatedDate: 2024-09-03
+---
+
+You can now use the AWS [Application Migration Service](https://docs.aws.amazon.com/mgn/latest/ug/what-is-application-migration-service.html) to install the New Relic agent on applications migrating into AWS.
+
+## Setup [#setup]
+
+To have AWS install the [New Relic Infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux/), Create a Post-launch template through the AWS console following these steps:
+
+1. Open Application Migration Services from your [AWS console](https://console.aws.amazon.com/mgn/home)
+2. Open the desired Source server from the available list
+3. Select the `Post-launch settings` tab
+4. Select `Edit`, and add the `Install New Relic agent` action, and add in the expected variables 
+
```

Comment:
```suggestion
1.  from your [AWS console](https://console.aws.amazon.com/mgn/home), open <DNT>**Application Migration Services** </DNT>.
2. Open the desired source server from the available list.
3. Click the **Post-launch settings** tab
4. Select **Edit**, add the `Install New Relic agent` action, and add in the expected variables.

```

@jcountsNR What are the expected variables? I think we should document them here

Reply:
Unfortunately AWS doesn't let me view that point without having an actual application to migrate into AWS, and I don't really have one handy. Once you select the new relic agent option, it will require that you include the accountID and the License Key, but I don't know exactly what the variables are called

Reply:
@jcountsNR How about this:

 Select **Edit > Install New Relic agent** and add your account ID and license key.